build/cache.go

```diff
@@ -0,0 +1,60 @@
+package build
+
+import (
+	"encoding/csv"
+	"strings"
+
+	"github.com/moby/buildkit/client"
+	"github.com/pkg/errors"
+)
+
+func ParseCacheEntry(in []string) ([]client.CacheOptionsEntry, error) {
```

Comment:
can we import buildctl pkg rather than duplicating the code?

Reply:
It is a little different. No legacy mode and accepts`ref` and `ref,ref` for compatibility with existing docker syntax for `cache-from`.